### PR TITLE
Fix EZP-25565: Reimplement database system info collector

### PR DESCRIPTION
**Pull request by glye**
> Implements https://jira.ez.no/browse/EZP-25565, subtask of https://jira.ez.no/browse/EZP-25563
> Status: Ready for review

Collects information about the database in use.

- [x] squash before merge if needed

### CODE REVIEW

**Comment by andrerom** [comment]
+1

**Comment by glye** [comment]
Review ping @bdunogier @dpobel

File: SystemInfo/Collector/DatabaseSystemInfoCollector.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo\Collector;
+
+use Doctrine\DBAL\Connection;
+use EzSystems\EzSupportToolsBundle\SystemInfo\Value;
+
+/**
+ * Collects database information.
+ */
+class DatabaseSystemInfoCollector implements SystemInfoCollector
```
Comment (bdunogier) [inline SystemInfo/Collector/DatabaseSystemInfoCollector.php]: The implementation would actually be a `DoctrineDatabaseSystemInfoCollector`, would it not ?

**Comment by bdunogier** [comment]
The micro naming remark above, I'll be +1.

**Comment by glye** [comment]
DatabaseSystemInfoCollector renamed to DoctrineDatabaseSystemInfoCollector (including test)

**Pull request merged**